test/lib/rubycritic/core/analysed_modules_collection_test.rb

```diff
@@ -26,7 +26,7 @@
       let(:paths) { "test/samples/" }
 
       it "recursively registers all files" do
-        subject.count.must_equal 13
+        subject.count.must_equal 14
```

Comment:
This is a pita... we have to increment this if we add new sample files to test with...

What about we change to: `subject.count.must_equal Dir['test/samples/**/*.rb'].count`?